python/ray/workflow/api.py

```diff
@@ -89,6 +91,30 @@ def _ensure_workflow_initialized() -> None:
             init()
 
 
+def client_mode_wrap(func):
+    """Wraps a function called during client mode for execution as a remote task.
+
+    Adopted from "ray._private.client_mode_hook.client_mode_wrap"
```

Comment:
should we just import it from there?

Reply:
some changes are made (e.g., init the workflow instead of init Ray; the latter does not specify a storage)

Reply:
I'll add some comments in the code